cmd/cli/cli.go

```diff
@@ -13,12 +14,14 @@ var project string
 var branch string
 var publishReview bool
 var markAsPublished bool
+var reviewEvent string
 
 func init() {
 	CliCmd.PersistentFlags().StringVar(&project, "project", "my-project", "Sonarqube project name")
 	CliCmd.PersistentFlags().StringVar(&branch, "branch", "my-branch", "SCM branch name")
 	CliCmd.PersistentFlags().BoolVar(&publishReview, "publish", false, "Publish review in the SCM")
 	CliCmd.PersistentFlags().BoolVar(&markAsPublished, "mark", false, "Mark the issue as published to avoid sending it again")
+	CliCmd.PersistentFlags().StringVar(&reviewEvent, "review-event", scm.REVIEW_EVENT_REQUEST_CHANGES, "GitHub review event type")
```

Comment:
I am thinking whether it wouldn't be better to have just boolean flag 🤔

Reply:
hmm, I would keep it as it is as with boolean flag it would make it less readable, for example `request-changes-event=true`, but it is unclear what will happen in false scenario (can be in the docs but I don't know...)